src/components/ShareStructure/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as ShareStructure } from './ShareStructure.vue'
+export { default as EditShareStructure } from './EditShareStructure.vue'
```

Comment:
Do you need to also include the currency lookup mixin here?

Reply:
Or not, if it's not needed anywhere else atm?

Reply:
It isn't used anywhere else atm, so i believe we are good without exporting for the time being.